### ISSUE TITLE
#214 broke Docker runtime

### ISSUE THREAD

**Issue by oliverhu** [linkedin/TonY#299]
```
-            String envDockerImage = (String) dockerRuntimeClass.getField(Constants.ENV_DOCKER_CONTAINER_TYPE).get(null);
+            String envDockerImage = (String) dockerRuntimeClass.getField(Constants.ENV_DOCKER_CONTAINER_IMAGE).get(null);```

**Comment by oliverhu** [comment]
there is a typo above

**Comment by oliverhu** [comment]
fixed in #300
